src/middlewares/validator.js

```diff
@@ -0,0 +1,32 @@
+import errorRes from '../helpers/successHandler';
```

Comment:
why are we importing the errorRes from the sucessHandler? shouldn't it be imported from errorHandler? please check on that

Reply:
Done thanks